modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1039,6 +1039,129 @@ public void testAddObjectEntry() throws Exception {
 			modifiableSystemObjectDefinition.getObjectDefinitionId());
 	}
 
+	@FeatureFlags("LPD-32050")
+	@Test
+	public void testAddObjectEntryAfterRecreatingLocalizedObjectField()
```

Comment:
I would rename this test to `testAddObjectEntryWithLocalizedObjectField()`.

However, there are 2 scenarios of `LocalizedObjectField` in the `testAddObjectEntry()` method. There is a chance that Brian will notice that and complain about this inconsistency.

-  testAddObjectEntryWithLocalizedObjectField()
// creation of dynamicObjectDefinitionLocalizationTable
// modifiable System Object Definition
// site scoped

Reply:
We discussed and we are only renaming the test method to `testAddObjectEntryAfterDeletingLocalizedObjectField()`